packages/gateway/package.json

```diff
@@ -20,8 +20,10 @@
   },
   "dependencies": {
     "itty-router": "^2.4.5",
+    "multicodec": "^3.2.1",
```

Comment:
This package is deprecated, but it seems to be the only way to get name for a codec...